### PR TITLE
feat: add formatter

### PR DESCRIPTION
**Pull request by kamadorueda**
- Run `yarn run format` to format files locally, and `yarn run check-format` to check that files are properly formatted on the CI

Signed-off-by: Kevin Amado <kamadorueda@gmail.com>

### CODE REVIEW

File: server/src/server.ts [hunk 1]
```diff
@@ -1,11 +1,8 @@
 import { App } from "./app";
 import { CalendarController } from "./controllers/calendar.controller";
 
-
 const port: number = 3000;
```
Comment (cloudyyoung) [inline server/src/server.ts:4]: Could you change port to `5000`? It conflicts with the frontend port.

**Review by cloudyyoung — APPROVED** [review]
LGTM! Just one comment

File: server/src/server.ts (hunk 1 above)
Comment (kamadorueda) [reply server/src/server.ts]: It's been changed to 3001 in another commit so we shouldn't have the collision anymore

**Review by tim-macphail — APPROVED** [review]
Fixed gitignore conflict, LGTM

**Comment by kamadorueda** [comment]
Take into account that this only formats the server, for now, adding it to the client would be a similar thing

**Pull request merged**